python/mxnet/numpy_op_signature.py

```diff
@@ -19,6 +19,7 @@
 
 import sys
 import warnings
+import inspect
```

Comment:
Thanks for putting the efforts on taming pylint. I'm curious why this reorganization of the code would make the pylint error disappear?

Reply:
So, first I should mention that as I developed the PR on my local machine, I was seeing slightly different pylint output than the upstream CI.  This could be due to a different pylint version.  In the comment that follows, I'll refer to the pylint command I posted above in the PR description as `<pylint_cmd>`:

With the `import inspect` statement in its original location, I saw:
```
$ <pylint_cmd> | grep numpy_op_signature
************* Module mxnet.numpy_op_signature
python/mxnet/numpy_op_signature.py:C0415:63,4: _register_op_signatures: Import outside toplevel (inspect)
```
With the `import inspect` statement moved, no such output line was generated.  Since I was touching the file numpy_op_signature.py anyway to eliminate the E0402 error, I thought I'd make the file totally clean w.r.t pylint by moving the `import inspect`.

Introducing the line `op_module = root_module` was the fix that really mattered (eliminated the E0402 error).  Since it didn't change the logic of the code materially, I view this as working around the quirks of numpy.

Reply:
Indeed, pylint is pinned in some cases inside the ci folder:

install/requirements
30:pylint==2.3.1; python_version >= '3.0'

We should use just one version of pylint...

Reply:
CI appears to pull in pylint 2.3.1.  The most current pylint is 2.4.2.